ruby-challenges.rb

```diff
@@ -10,25 +18,84 @@
 reposts3 = 221
 # Expected output: '221 is odd'
 
+def even_odd num
+    if num % 2 == 0
+        "This is an even number"
+    elsif num % 2 != 0 
+        "This is an odd number"
+    else 
+        "This aint even a number dude"
+    end
+end
 
+# p even_odd reposts1
+# # "This is an odd number"
+# p even_odd reposts2
+# # "This is an even number"
+# p even_odd reposts3
+# # "This is an odd number"
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. 
 # HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
+# Puesdo C:
+# Define a new method called vowel_remover
+# Give the method one parameter 'string' to match the given inputs
+# In the method create a new variable called new_string. Inside store the string parameter with .downcase attached to make all letters the same in the string.
+# Also ad the .delete method to get rid of any vowels within that string.
+# return the new_string variable with .split and .map attached. This will first split the string into individual letters and .map will iterate through each letter as an array.
+# Then use &:capitalize as the short hand to capitalize the words.
+# Last use .join to pull the array of letters back together as strings. 
+
 beatles_album1 = 'Rubber Soul'
 # Expected output: 'Rbbr Sl'
 beatles_album2 = 'Sgt Pepper'
 # Expected output: 'Sgt Pppr'
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
-
-
+def vowel_remover string
+   new_string = string.downcase.delete('aeiou')
+   new_string.split.map(&:capitalize).join(' ')
+end
+# p vowel_remover beatles_album1
+# # "Rbbr Sl"
+# p vowel_remover beatles_album2
+# # "Sgt Pppr"
+# p vowel_remover beatles_album3
+# # "Bby Rd"
```

Comment:
Again, remember to examine your input and output. Your solution of recapitalizing is very on-brand for Ruby! But in this case I would suggest updating the delete action to include uppercase letters. `string.delete('aeiouAEIOU')`